docs/source/guide/minibatch-prefetching.rst

```diff
@@ -3,94 +3,25 @@
 6.8 Feature Prefetching
```

Comment:
Consider change the title entirely, e.g. datapipe overlapping, and rewrite it to illustrate how we handles overlapping different stage with the advantage of new graphbolt datapipe mechanism.

Reply:
Shall we use "pipeline parallelism" or "pipelining" instead of overlapping or prefetching? It would fit the GPU graph sampling pipelining approach in our proposal better.